### ISSUE TITLE
cannot use shortcut (show-corrupted-measures)

### ISSUE THREAD

**Issue by jamestomk** [musescore/MuseScore#24998]
### Issue type

UI bug (incorrect info or interface appearance)

### Description with steps to reproduce

see Musescore help discussion https://musescore.org/en/comment/1261028
according to Jojo-Schmitz https://musescore.org/en/user/4901, this operation is commented out except for Debug build, and therefore the operation "show-corrupted-measures" should not appear in the production build Shortcuts list.

1. edit Preferences
2. choose Shortcuts
3. search for "show-corrupted-measures"
4. click on "Define"
5. set up an unused keystroke like "Ctl-,"
6. click on OK, notice that the short cut is accepted, click on OK to close
7. attempt to use the shortcut either by selecting a staff, measure, or select All and use the keystroke
8. notice nothing happens

### Supporting files, videos and screenshots

![cap](https://github.com/user-attachments/assets/84d67186-5804-413e-9102-7bdcd4288b52)

### What is the latest version of MuseScore Studio where this issue is present?

MuseScore Studio 4.4.2 64-bit | Windows

### Regression

Choose option...

### Operating system

Windows 11

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by Jojo-Schmitz** [comment]
And this seems to be a regression vs. Mu3

**Comment by Jojo-Schmitz** [comment]
Hold on, if you enable dev mode, that "show-corrupted-measures" should work, like when you have this in ypout MuseScore`4.ini (Wondows)
```
[application]
devModeEnabled=true
```
In which case you won't really need the shortcut, unless you want to switch the "Show corrupted measures" off, as it is on by default (So that shortuct should better be named "Toggle 'Show corrupted measures'")

**Comment by Jojo-Schmitz** [comment]
I don't know whether there's a documented official way of ernabling dev mode

**Comment by cbjeukendrup** [comment]
No, dev mode won't help here, `MStaff::corrupted` is surrounded with `#ifndef NDEBUG`

**Comment by Jojo-Schmitz** [comment]
OK, then the shortcut and the menu entry should get the same treatment as that `MStaff::corrupted` or, better, vice versa, as this is a really usefull option

**Comment by cbjeukendrup** [comment]
Yes, afaics, the overhead introduced by this option is very minimal, so we can provide it for non-debug builds too

**Comment by Jojo-Schmitz** [comment]
There's an Easer Egg:

Enabling dev mode is done by by opening the About dialog, then clicking the app icon there three times, and restarting the program. Note that there is no visual feedback after clicking the app icon thrice, until you restart.

Doing the same thing again switches it off again.
